src/test/resources/Hotels/Calendar.feature

```diff
@@ -0,0 +1,15 @@
+#TC17
+Feature: Disabled dates in calendar
+  Background:
+    Given I launch Hotels.com
+  Scenario: Verify past date and back button on Current month's calendar is disabled
+
+     When I click on Check-in Calendar
+     Then I verify past dates if any are disabled
+      And I verify back button on current month is disabled
+     When I click on Done button on calendar
+      And I click on Check-out Calendar
+    Then I verify past dates if any are disabled
+    And I verify back button on current month is disabled
```

Comment:
If lines are not alined, it doesn't look good. You select all lines then Code > Reformat code

![Screen Shot 2022-05-31 at 1 36 45 PM](https://user-images.githubusercontent.com/30092854/173193562-27c5401f-053e-4e8c-8b5d-7fc3e573c2fe.png)